README.md

```diff
@@ -971,11 +967,12 @@ A preferred way would be to select a default value using the `selected` attribut
     ).asAccessibleAutocomplete(Some(
         AccessibleAutocomplete(
             showAllValues = true,
-            autoSelect = false)
+            autoSelect = false,
+            emptyItem = Some("Select an item")
```

Comment:
hmm, I know people said no to placeholder - but I don't think it will be clear what this param will do - a bit of me is wondering seeing this, if we're not going to ever forward this to the placeholder param of a11y autocomplete, if we should even give people an option for this rather that just having it be guidance that you need to explicitly pass an empty value to create a initial placeholder option